code/procs/command_alert.dm

```diff
@@ -67,7 +67,7 @@
 				text = replacetext(text, "%mrand_name%", rand_mob_mult.name)
 				text = replacetext(text, "%mrand_job%", rand_mob_mult.job ? rand_mob_mult.job : "space hobo")
 
-				boutput(M, "<h1 class='alert'>[command_name()] Update</h1>")
+				boutput(M, "<h1 class='alert'>[ALERT_GENERAL]</h1>")
```

Comment:
did you mean to remove Update

Reply:
I didn’t, the new alerts contains update/respective titles

Reply:
then why didn't you remove it everywhere else

Reply:
Oh?? Hold on I’ll need to peep? It’s late rn but I thought it was removed on all the others but I may be misremembering.

Reply:
Oh yep I entirely missed that for this PR ahahaha. Good catch, I’ll fix it tomorrow